{
  "id": "EUVD-2024-22158",
  "enisaUuid": "65948fb6-eaaa-38da-81f8-ee2942181f90",
  "description": "Remote Code Execution with untrusted URI of UDF vulnerability in Apache IoTDB. The attacker who has\u00a0privilege to create UDF can register malicious function from\u00a0untrusted URI.\n\nThis issue affects Apache IoTDB: from 1.0.0 before 1.3.4.\n\nUsers are recommended to upgrade to version 1.3.4, which fixes the issue.",
  "datePublished": "May 14, 2025, 10:42:20 AM",
  "dateUpdated": "May 15, 2025, 4:01:59 AM",
  "baseScore": 9.8,
  "baseScoreVersion": "3.1",
  "baseScoreVector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H",
  "references": [
    "https://nvd.nist.gov/vuln/detail/CVE-2024-24780",
    "https://lists.apache.org/thread/xphtm98v3zsk9vlpfh481m1ry2ctxvmj",
    "http://www.openwall.com/lists/oss-security/2025/05/14/2"
  ],
  "aliases": [
    "CVE-2024-24780",
    "GSD-2024-24780"
  ],
  "assigner": "apache",
  "epss": 0.29,
  "enisaIdProduct": [
    {
      "id": "66340c21-5746-35e0-b5e8-59d7f558235b",
      "product": {
        "name": "Apache IoTDB"
      },
      "product_version": "1.0.0 <1.3.4"
    }
  ],
  "enisaIdVendor": [
    {
      "id": "edbd2dd0-acf5-3617-a1c5-6399e26da637",
      "vendor": {
        "name": "Apache Software Foundation"
      }
    }
  ],
  "cve": "CVE-2024-24780"
}